client/assets/js/sprites/Player.js

```diff
@@ -195,24 +213,29 @@ class Player extends ArcadeSprite {
      *
      */
     createCrate(){
-        if (this.itemCooldown > 0) {
-            return;
-        }
-
-        this.itemCooldown = 20;
-
-        let crate = new Crate(this.scene, this.x, this.y)
-
-        crateGroup.add(crate)
+        // Create new crate and add to group
+        crateGroup.add(new Crate(this.scene, this.x-10, this.y))
+
+        //collision box to eachother
+        this.scene.physics.add.collider(crateGroup, crateGroup, function (s1, s2) {
```

Comment:
Zorgt deze collision callback er voor dat crates niet meer door elkaar heen vallen? Of is er een andere reden voor? Ik *denk* dat we dat ook met bepaalde properties op de body kunnen fixen (das beter voor performance dan een callback), maar weet t niet zeker.

Reply:
Ja ik heb deze fix van labs.phaser.com. Dit was inderdaad de fix dat als we een hogere stapel hadden dan 3 dat ze door elkaar heen vielen

Reply:
denk dat we toch n keer matter physics moeten pakken dan. arcade physics is een beetje nep physics lijkt t :P